src/App.js

```diff
@@ -13,18 +13,15 @@ class App extends React.Component {
         this.state = {
             dataCard: {
                 statuses: null
-            },
-            modalWindow: {
-                visible: false                
-            }     
+            }                 
         };  
```

Comment:
@DmitryApan you need to initialize `idCard` in state

Reply:
Yes